### DISCUSSION TITLE
Encuesta #1

### DISCUSSION THREAD

**Discussion by Ballwictb** [IsraC0d33/Proyecto1Marcas#10] [Polls]
Buenas equipo.

Sabréis que últimamente @IsraC0d33 esta intentando implementar el nuevo workflow de <a href="https://platzi.com/blog/sonarcloud-mejora-codigo-sast/">SonarCloud</a>.
A conseguido arreglar unos cuantos errores pero no llega a ejecutarse de forma correcta.

Hoy **_07/02/2024_** me a comentado su descontento y la posibilidad de prescindir del workflow.

Hoy lo sometemos a votación.

**INDICACIONES** : 
-La votación no termina hasta que voten todos los miembros.
-Podéis plantear soluciones alternativas en el apartado de ideas (si hay una buena alternativa se dará por concluida la votación)
-Podéis comentar cosas al respecto en el foro.

**AVISO**

**_Cualquier comentario/mensaje que incumpla los <a href="https://github.com/IsraC0d33/Proyecto1Marcas/blob/a9c5e96814f7f9c19dbb6f89a8246c94207d17da/CODE_OF_CONDUCT.md">Codigos de Conducta</a> quedaran sujetos a las Directrices de Aplicación_**

**Comment by IsraC0d33** [comment]
el que vote si le doy un besito

**Comment by MuiGt** [comment]
Se supone que el propio GitHub tiene un compilador de errores, ¿ese no nos sirve?
Y además, ¿no podemos usar otro compilador?

**Reply by Ballwictb** [reply]
GitHub tiene un compilador de errores por defecto??

Que yo sepa no, por eso existen las posibles integraciones para el análisis de código.

Una idea que se me viene aun que un poco mas compleja seria que ya en nuestro IDE tengamos alguna extensión que nos ayude a comprobar la calidad del código o que no tenga errores.

Por ejemplo <a href="https://cspell.org/">Spell</a>

Lo complejo viene a la hora de que:

1. Todos tengamos en Visual Studio Code instalada la extensión Spell


En cuanto a la configuración no habría problemas ya que crearía una configuración general dentro de GitHub.

Así cuando alguien haga un codespace o un git clone ya tendría ahí la configuración general. 

Seria como por ejemplo con editorconfig el cual nos ayuda a tener una configuración básica correcta/establecida.

**Comment by Ballwictb** [comment]
Arreglado
